amazon-kinesis-sql-connector-flink/pom.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+This file has been modified from the original.
+
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>software.amazon.kinesis</groupId>
+		<artifactId>amazon-kinesis-connector-flink-parent</artifactId>
+		<version>2.1.0-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+
+	<name>Amazon Kinesis SQL Connector for Apache Flink</name>
+	<artifactId>amazon-kinesis-sql-connector-flink</artifactId>
+	<packaging>jar</packaging>
+
+	<dependencies>
+		<dependency>
+			<groupId>software.amazon.kinesis</groupId>
+			<artifactId>amazon-kinesis-connector-flink</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+	</dependencies>
+
+	<dependencyManagement>
```

Comment:
Whatis the point of having `flink-test-utils-junit` defined in `dependencyManagement` and not in `dependencies`?

Reply:
It is not a dependency, there are no unit tests in this module (or code). I added it because it is being added somewhere in a parent pom and resolving to use the project version (2.1.0-SNAPSHOT). Without adding this the build fails. Since it is a dependency management block, it is not pulled into the dependency chain (it is used to manage the version should it be included transitively).